README.md

```diff
@@ -15,15 +15,15 @@ Here are the principles we use to guide our development.  From [opensearch.org](
 >
 >**1. Great software**. If it doesn’t solve your problems, everything else is moot. It’s going to be software you love to use.
 >
->**2. Open source like we mean it**. We are invested in this being a successful open source project for the long term. It’s all Apache 2.0. There’s no Contributor License >Agreement. Easy.
+>**2. Open source like we mean it**. We are invested in this being a successful open source project for the long term. It’s all Apache 2.0. There’s no Contributor License Agreement. Easy.
 >
->**3. A level playing field**. We will not tweak the software so that it runs better for any vendor (including AWS) at the expense of others. If this happens, call it out and we >will fix it as a community.
+>**3. A level playing field**. We will not tweak the software so that it runs better for any vendor (including AWS) at the expense of others. If this happens, call it out and we will fix it as a community.
 >
 >**4. Used everywhere**. Our goal is for as many people as possible to use it in their business, their software, and their projects. Use it however you want. Surprise us!
 >
 >**5. Made with your input**. We will ask for public input on direction, requirements, and implementation for any feature we build.
 >
->**6. Open to contributions**. Great open source software is built together, with a diverse community of contributors. If you want to get involved at any level - big, small, or >huge - we will find a way to make that happen. We don’t know what that looks like yet, and we look forward to figuring it out together.
+>**6. Open to contributions**. Great open source software is built together, with a diverse community of contributors. If you want to get involved at any level big, small, or huge we will find a way to make that happen. We don’t know what that looks like yet, and we look forward to figuring it out together.
```

Comment:
TL;DR: Replace the hyphens with em dashes in the second sentence instead of removing them entirely.

I believe the previous grammar in this sentence was nearly correct. If we replace the hyphens with em dashes (—) and remove the extra spaces (i.e. ..."level—big, small, or huge—we"...), then it's grammatically correct. The em dashes separate the phrase that specifies the list of levels—big, small, or huge—from the rest of the sentence. Without separating the phrases with commas, parentheses, or something else, the sentence doesn't read very well.

Source: my faint memory of 10th grade high school English, and [Grammarist](https://grammarist.com/grammar/emdash/)

Reply:
Wow improve my grammar while improve coding. I checked the em dash usage and I think it fits here well.